lab-miketreat/test/module-test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
Also this isn't invoking your helper function, just the fs module. Per instructions, you need to be invoking your helper function.